### PR TITLE
Fishing-themed Escape Shuttle

### PR DESCRIPTION
**Pull request by Jacquerel**
## About The Pull Request

I can't do much coding until you review my other PRs so I'm making a mapping PR instead.
I actually made this a while ago while I was trying out strongDMM. It turns out: it's a good tool and easy to use.

This mid-tier shuttle isn't enormous and is shaped like a fish. It dedicates much of its internal space to an artificial fishing environment, plus fishing equipment storage. Plus look at that lovely wood panelling!
There's not a lot of seating or a large medbay, but there's five fishing rods for people to wrestle each other over plus some aquariums to store your catches in.

It contains a variety of fishing biomes (ocean, moisture trap, hole, portal) but I couldn't fit "lava" in there even though I wanted to because it's hardcoded to only have fish in it on the mining z-level.
If you're very lucky and nobody shoves you, the time between the shuttle docking at the station and arriving at Centcomm might be enough time for you to catch maybe four entire fish. Wow!

## Why It's Good For The Game

There are plenty of novelty shuttle options but I think this one is good for a personal touch of "the Captain would rather be fishing than hearing you complain about the nuclear operatives".

## Changelog

:cl:
add: Tell your crew how much you care by ordering a shuttle where half of the seats have been removed so that you can get some angling done before you clock out.
/:cl:

### CODE REVIEW

**Comment by Sealed101** [comment]
how many fishing rods are there in total

**Comment by Jacquerel** [comment]
> how many fishing rods are there in total

There's two in the main room and three more in lockers

**Comment by Jackal-boop** [comment]
please give it a bass pro shop

File: _maps/shuttles/emergency_fish.dmm [hunk 1]
```diff
@@ -0,0 +1,1447 @@
...
+	},
+/turf/open/floor/wood/tile,
+/area/shuttle/escape)
+"rj" = (
+/obj/docking_port/mobile/emergency{
+	name = "Angler's Choice emergency shuttle";
+	height = 13;
+	width = 27;
+	dwidth = 6
+	},
```
Comment (ZephyrTFA) [inline _maps/shuttles/emergency_fish.dmm:385]:
```suggestion
	name = "Angler's Choice emergency shuttle"
	},
```

Comment (Jacquerel) [reply _maps/shuttles/emergency_fish.dmm:385]: When I wrote this (a few months ago) I had to make this edit or the brig disappeared
I'm glad that sems to have been fixed in the intervening time

**Comment by OrionTheFox** [comment]
Screenshot_20221207-151701_Chrome
This doesnt match the top, assume its not intentional?

**Review by san7890 — CHANGES_REQUESTED** [review]
image

I don't have time to fully review this (plus i let feature mapping PRs linger a bit so more eyes from the community can give feedback on it), but can you please please please dir-flip these stasis beds the other way? thank you

just mark "ready for review" when this is addressed

**Comment by Jacquerel** [comment]
> image
> 
> I don't have time to fully review this (plus i let feature mapping PRs linger a bit so more eyes from the community can give feedback on it), but can you please please please dir-flip these stasis beds the other way? thank you
> 
> just mark "ready for review" when this is addressed

Done & updated screenshot

**Comment by ArcaneMusic** [comment]
But are the doctors, fish doctors, jacq.

**Review by Time-Green — APPROVED** [review]
I'm not a maptainer but this has been pretty good on the rounds I've played with it testmerged 👍

**Pull request merged**